nodes/host/host.go

```diff
@@ -44,11 +45,15 @@ func (n *host) Init(cfg *types.NodeConfig, opts ...nodes.NodeOption) error {
 	n.Cfg.IsRootNamespaceBased = true
 	return nil
 }
-func (*host) Deploy(_ context.Context, _ *nodes.DeployParams) error { return nil }
-func (*host) GetImages(_ context.Context) map[string]string         { return map[string]string{} }
-func (*host) PullImage(_ context.Context) error                     { return nil }
-func (*host) Delete(_ context.Context) error                        { return nil }
-func (*host) WithMgmtNet(*types.MgmtNet)                            {}
+func (n *host) Deploy(_ context.Context, _ *nodes.DeployParams) error {
+	n.SetState(state.Deployed)
+	return nil
+}
```

Comment:
@steiler I wonder if setting state here is needed, given that GenericLinkNode sets this state to deployed by default?

Reply:
It is a little tricky, but the fakehostnode is a different struct, host and fakehostnode might exist at the same time in the nodes.Node. So from that perspective it is needed.